### PR TITLE
Working Recycler view swipe on Calendar fragment

### PR DESCRIPTION
**Pull request by pranitishettiwar**
ToDo: Delete completed tasks from firebase

demo

### CODE REVIEW

File: app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java [hunk 1]
```diff
@@ -83,6 +86,10 @@
     public CalendarFragment() {
     }
 
+    public static List<Task> getmCompletedTasks() {
```
Comment (statham) [inline app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java]: this should either be `getMCompletedTasks` to keep with camel case conventions or `getCompletedTasks`. I like the latter, as it abstracts knowledge of how we actually store the tasks in a variable from other functions that have to call this

File: app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java [hunk 2]
```diff
@@ -139,6 +146,38 @@ private void setupRecyclerView(View view) {
...
+            @Override
+            public void onSwiped(RecyclerView.ViewHolder viewHolder, int direction) {
+                completedTaskList(viewHolder.getAdapterPosition());
+            }
+
+        });
+        itemTouchHelper.attachToRecyclerView(mRvTasks);
+    }
+
+    private void completedTaskList(int position) {
```
Comment (statham) [inline app/src/main/java/com/kyphipraniti/fitnesstasks/fragments/CalendarFragment.java]: i think this should be `completeTask` yeah? looks like it's being used to mark a task as complete.

**Review by statham — CHANGES_REQUESTED** [review]
This looks great! I just have a couple small changes, and I'm also wondering if it makes sense to be able to reorder the tasks if we're not also updating the datetime that they are due?

**Comment by pranitishettiwar** [comment]
@statham I will address those changes. I added the drag and reorder functionality, as it can be cool to demo. We can probably later decide if we wanna keep or remove.

**Comment by pranitishettiwar** [comment]
Changes done, Merging to master.

**Pull request merged**